packages/spruce-skill/interface/pages/skill-views/example.js

```diff
@@ -52,6 +52,31 @@ class TestSkillView extends React.Component<Props> {
 										}
 									})
 								}}
+							/>{' '}
```

Comment:
what's this space here for? I think you'll want to wrap these buttons in a `<ButtonGroup>` to space the buttons appropriately

Reply:
Sure, done